test/lib/PrintEquality.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <iostream>
+#include <cxxabi.h>
```

Comment:
We need at least `<string>` here as well.

Reply:
🤔 weird that its compiling fine for this?

Reply:
It's compiling fine because you only use this file here:
https://github.com/Expensify/Bedrock/pull/1009/files#diff-0049b41cb8098a521abb593b7921df73b67adeadfa658315a972292110895479R32

And that's below an include of `<string>` in that file. It will stop compiling if someone removes that include from tpunit++.hpp, or includes this file in some other place that doesn't include `string`.

Reply:
Ah gotcha. Makes sense. Added.